tools/rebuild_provision_request.rb

```diff
@@ -1,6 +1,6 @@
 require File.expand_path('../config/environment', __dir__)
 require 'trollop'
-require 'rest-client'
+require 'rest_client'
```

Comment:
We use the `rest-client` gem.  The underscored name is relying on backward compatibility in the gem.   

See https://github.com/rest-client/rest-client/blob/master/lib/rest_client.rb#L1